docs/pages/developer-guide.md

```diff
@@ -1946,14 +1950,45 @@ To be clear, the config templating provided by the `bootstrap` tool may be appli
 
 ### Task Environment
 
-While some environment variables are included by default in each task as a convenience, you may also specify custom environment variables yourself.  For example, a typical task would include the following key/value pairs in its environment:
+Task configuration is generally exposed using environment variables. A number environment variables describing the task and/or cluster environment are provided automatically, while others may be manually specified by the developer.
```

Comment:
"A number of environment variables that describe the task and/or cluster environment are provided automatically, while the developer can manually specify others."

Reply:
Done